tests/fixture_valid.js

```diff
@@ -192,47 +129,41 @@ module.exports.checkFixture = function checkFixture(fixture, usedShortNames=[])
       }
 
       if ('capabilities' in channel) {
-        for (let i=0; i<channel.capabilities.length; i++) {
-          const cap = channel.capabilities[i];
+        for (let i = 1; i <= channel.capabilities.length; i++) {
+          const cap = channel.capabilities[i-1];
 
+          // range valid / no overlapping
           if (cap.range[1] >= dmxMaxBound) {
             result.errors.push({
               description: `range values must be strictly less than ${dmxMaxBound} in capability #${i} in channel '${ch}'.`,
               error: null
             });
-            break;
```

Comment:
If one range is invalid, the later tests can not rely on it anymore and are therefore not meaningful (and in the worst case misleading). So better just exit the test and potentially hide some errors that will then only occur on retesting.